src/components/organisms/Navbar/Navbar/Navbar.test.tsx

```diff
@@ -60,27 +61,34 @@ describe('<Navbar />', () => {
     },
   ];
 
-  it('should render small device navigation with default props', () => {
+  it('should not render the hamburger icon when no content is added', () => {
+    const { queryByTestId } = render(<Navbar />);
+
+    const hamburgerIcon = queryByTestId('hamburger-icon');
+    expect(hamburgerIcon).not.toBeInTheDocument();
+  });
+
+  it('should render small device navigation with default props with no a11y violations', async () => {
     const { container } = render(<Navbar links={NAV_ITEMS} />);
     expect(container).toMatchSnapshot();
-  });
 
-  it('should not render the hamburger icon when no content is added', () => {
-    render(<Navbar />);
-    const hamburgerIcon = screen.queryByTestId('hamburger-icon');
-    expect(hamburgerIcon).not.toBeInTheDocument();
+    const results = await axe(container.innerHTML);
+    expect(results).toHaveNoViolations();
```

Comment:
This won't pass until https://github.com/zopaUK/react-components/pull/689 is merged.